examples/org.eclipse.glsp.example.workflow/src/org/eclipse/glsp/example/workflow/utils/WorkflowBuilder.java

```diff
@@ -126,20 +124,14 @@ public void setProperties(final TaskNode taskNode) {
          taskNode.setName(name);
          taskNode.setTaskType(taskType);
          taskNode.setDuration(duration);
-         taskNode.setLayout(GConstants.Layout.VBOX);
-         taskNode.getChildren().add(createCompartment(taskNode));
```

Comment:
Due to the change of the task node layout, the validation of task nodes regarding the first character to be upper-case (i.e. [`validateTaskNode_labelStartsUpperCase`](https://github.com/eclipse-glsp/glsp-server/blob/b4dbb5796fed14de0f33b6e3da5168b9247fefa5/examples/org.eclipse.glsp.example.workflow/src/org/eclipse/glsp/example/workflow/marker/WorkflowModelValidator.java#L84) does not work as expected anymore. Could you please adapt this validation method?

Reply:
Nice catch! Fixed with 9c1d1e3